src/QirRuntime/build.py

```diff
@@ -1,102 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why are these files removed?

Reply:
The .py files are not applicable any more.
The .ps1 files are used for building and testing.